app/views/transaction/show.html.erb

```diff
@@ -110,9 +110,10 @@
                   margin_bottom: true,
                   data_attributes: data_attributes %>
       <% if hmrc_temporary_ab_test_variant_b? %>
-        <div class="govuk-body govuk-!-padding-bottom-5">
-          <%= "#{t('ab_testing.ready_reckoner_video_test_html')}".html_safe %>
-        </div>
+        <%= render "govuk_publishing_components/components/govspeak", {} do %>
+          <p><%= t('ab_testing.ready_reckoner_video_description') %></p>
+          <p><a href="https://www.youtube.com/watch?v=xHn31myAkio">How do I budget for my Self Assessment tax bill?</a></p>
```

Comment:
I was just wondering if we should put the href and link title in the locale file as well?

Reply:
Good shout, will look into.